hardhat.config.ts

```diff
@@ -76,221 +76,221 @@ const config: HardhatUserConfig = {
-      accounts,
-      chainId: 128,
-      live: true,
-      saveDeployments: true,
-    },
-    "heco-testnet": {
-      url: "https://http-testnet.hecochain.com",
-      accounts,
-      chainId: 256,
-      live: true,
-      saveDeployments: true,
-      tags: ["staging"],
-      gasMultiplier: 2,
-    },
-    avalanche: {
-      url: "https://api.avax.network/ext/bc/C/rpc",
-      accounts,
-      chainId: 43114,
-      live: true,
-      saveDeployments: true,
-      gasPrice: 470000000000,
-    },
-    fuji: {
-      url: "https://api.avax-test.network/ext/bc/C/rpc",
-      accounts,
-      chainId: 43113,
-      live: true,
-      saveDeployments: true,
-      tags: ["staging"],
-      gasMultiplier: 2,
-    },
-    harmony: {
-      url: "https://api.s0.t.hmny.io",
-      accounts,
-      chainId: 1666600000,
-      live: true,
-      saveDeployments: true,
-    },
-    "harmony-testnet": {
-      url: "https://api.s0.b.hmny.io",
-      accounts,
-      chainId: 1666700000,
-      live: true,
-      saveDeployments: true,
-      tags: ["staging"],
-      gasMultiplier: 2,
-    },
-    okex: {
-      url: "https://exchainrpc.okex.org",
-      accounts,
-      chainId: 66,
-      live: true,
-      saveDeployments: true,
-    },
-    "okex-testnet": {
-      url: "https://exchaintestrpc.okex.org",
-      accounts,
-      chainId: 65,
-      live: true,
-      saveDeployments: true,
-      tags: ["staging"],
-      gasMultiplier: 2,
-    },
-    arbitrum: {
-      url: "https://arb1.arbitrum.io/rpc",
-      accounts,
-      chainId: 42161,
-      live: true,
-      saveDeployments: true,
-      blockGasLimit: 700000,
-    },
-    "arbitrum-testnet": {
-      url: "https://kovan3.arbitrum.io/rpc",
-      accounts,
-      chainId: 79377087078960,
-      live: true,
-      saveDeployments: true,
-      tags: ["staging"],
-      gasMultiplier: 2,
-    },
-    celo: {
-      url: "https://forno.celo.org",
-      accounts,
-      chainId: 42220,
-      live: true,
-      saveDeployments: true,
-    },
-    palm: {
-      url: "https://palm-mainnet.infura.io/v3/da5fbfafcca14b109e2665290681e267",
-      accounts,
-      chainId: 11297108109,
-      live: true,
-      saveDeployments: true,
-    },
-    "palm-testnet": {
-      url: "https://palm-testnet.infura.io/v3/da5fbfafcca14b109e2665290681e267",
-      accounts,
-      chainId: 11297108099,
-      live: true,
-      saveDeployments: true,
-      tags: ["staging"],
-      gasMultiplier: 2,
-    },
-    moonriver: {
-      url: "https://rpc.moonriver.moonbeam.network",
-      accounts,
-      chainId: 1285,
-      live: true,
-      saveDeployments: true,
-    },
-    fuse: {
-      url: "https://rpc.fuse.io",
-      accounts,
-      chainId: 122,
-      live: true,
-      saveDeployments: true,
-    },
-    clover: {
-      url: "https://rpc-ivy.clover.finance",
-      accounts,
-      chainId: 1024,
-      live: true,
-      saveDeployments: true,
-    },
+    // ropsten: {
```

Comment:
@GooGrand in such cases I suggest using /* */ notation for comments to avoid large unnecessary diffs

Reply:
Also a very bad side-effect of this - we loose history, blame on those lines doesn't show real commits history, so it should be avoided